### DISCUSSION TITLE
OpenDKIM message every second

### DISCUSSION THREAD

**Discussion by fxp1303** [modoboa/modoboa#2166] [General]
Mail log is logging OpenDKIM message every second;

Feb 10 04:30:03 mail opendkim[2430]: D30D061100: no signing table match for 'root@mail.domain.net'
Feb 10 04:30:03 mail opendkim[2430]: D30D061100: no signature data

What setting do I need to implement to prevent these messages?

**Comment by tonioo** [comment]
Looks like your system tries to send a message to root@ every second, you should first try to understand why.
